src/MongoDB.Driver.Core/Core/Operations/ChangeStreamCursor.cs

```diff
@@ -76,6 +113,16 @@ public void Dispose()
             }
         }
 
+        /// <inheritdoc/>
+        public BsonDocument GetResumeToken()
+        {
+            return 
+                _postBatchResumeToken ?? 
```

Comment:
these lines together with `ChangeStreamOperation.GetInitialPostBatchResumeTokenIfRequired` satisfies to the unit tests result which are proposed for this ticket.